features/step_definitions/networking.rb

```diff
@@ -1590,3 +1590,12 @@
   end
   logger.info "The egressfirewall type is applied."
 end
+
+Given /^the cluster is dual stack network type$/ do
+  ensure_admin_tagged
+  @result = admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.serviceNetwork}")
+  unless @result[:response].count(":") >= 2 && @result[:response].count(".") >= 2  
+    logger.info "the cluster is not dual stack cluster, will skip this scenario"
```

Comment:
@zhaozhanqi Would you mind to bump up the log level to `warn` ?

Reply:
@liangxia `info` level cannot be shown in CI logs? since most of logs are using `info`

Reply:
There are too many info logs.  `skip_this_scenario` is an important info, we do not want team members to missed them and report issues why some of the tests are not executed.
Also if you search existing usage, you will see all the skip_this_scenario are using logger.warn
```
$ grep -r skip_this_scenario -B1
```

Reply:
ah.  ok. fixed.